common/types/product/TProductCreate.ts

```diff
@@ -1,11 +1,11 @@
 import TImage from '../TImage';
+import TProduct from './TProduct';
 
-type TProductCreate = {
-  title: string;
-  content?: string;
-  price: number;
+type TProductCreate = Pick<
+  TProduct,
+  'title' | 'content' | 'price' | 'categoryId'
+> & {
   images?: TImage[];
```

Comment:
```suggestion
> & {
  images?: TImage[];
```
이 부분은 앞에 Pick으로 지정한 타입과 images를 같이 쓰기 위해 & 을 넣으신게 맞나요?

Reply:
네 맞습니다!!